hibernate-reactive-core/src/main/java/org/hibernate/reactive/vertx/VertxInstance.java

```diff
@@ -24,7 +24,7 @@
  * <pre>{@code
  * new ReactiveServiceRegistryBuilder()
  *     .applySettings( properties )
- *     .addService( VertxInstance.class, () -> myVertx )
+ *     .addService( VertxInstance.class, (VertxInstance) () -> myVertx )
```

Comment:
@DavideD what's going on here? Why is this typecast now necessary?

Reply:
When I tried this code I had a compilation error and that was the only way of making it work.
Did I miss something?

Reply:
Oh so you're saying it was always necessary and my Javadoc code example was wrong?

Reply:
Yep

Reply:
OK. NP